### PR TITLE
beast2:  add beagle-lib and gromacs for beast2 support arm

### PR DESCRIPTION
**Pull request by GuangJie1**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/migrations/arch_rebuild.txt [hunk 1]
```diff
@@ -36,6 +36,7 @@ bayes-filters-lib
 bayeswave
 bazel
 bcrypt
+beagle-lib
```
Comment (martin-g) [inline recipe/migrations/arch_rebuild.txt:39]: `beagle-lib` is a Bioconda recipe. There is no feedstock for it at conda-forge.

Comment (GuangJie1) [reply recipe/migrations/arch_rebuild.txt:39]: This branch has been merged to master, do i need to delete the beagle-lib ?

Comment (martin-g) [reply recipe/migrations/arch_rebuild.txt:39]: Yes, please open a new PR to remove it!

Comment (GuangJie1) [reply recipe/migrations/arch_rebuild.txt:39]: ok, thank you.

**Pull request merged**